docs/source/install.rst

```diff
@@ -30,13 +44,32 @@ i.e. if you are contributing back to this project::
 That's it! For information on using ``repo2docker``, see
 :ref:`usage`.
 
-Note about Windows support
---------------------------
+.. _windows:
 
-Windows support by ``repo2docker`` is still in the experimental stage. 
+Windows support
+---------------
+
+Windows support by ``repo2docker`` is still in the experimental stage.
 
 An article about `using Windows and the WSL`_ (Windows Subsytem for Linux or
 Bash on Windows) provides additional information about Windows and docker.
 
 
 .. _using Windows and the WSL: https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly
+
+.. _jupyterhub:
+
+JupyterHub-ready images
+-----------------------
+
+`JupyterHub <https://jupyterhub.readthedocs.io/en/stable/>`_ allows multiple
+users to collaborate on a shared Jupyter server. ``repo2docker`` can build
+Docker images that can be shared within a JupyterHub deployment.  An example
+of using JupyterHub with ``repo2docker`` is Project Jupyter's free online
+service, `mybinder.org <https://mybinder.org>`_, which allows anyone to
+build a Docker image of a git repo online share the image with a URL.
```

Comment:
some missing words? I think we should try and spell things out so "repo" -> "repository". I think the missing words are: ... which allows anyone to share an executable version of a git repository by sharing a single link ?